tcf_website/templates/search/search.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base/base.html" %}
+{% load static %}
+
+{% block styles %}
+<link rel="stylesheet" href="{% static 'search/search.css' %}" />
+{% endblock %}
+
+{% block content %}
+<div class="container">
+    <h1>Search: {{query}}</h1>
+</div>
+
+<hr>
+
+<div class="container">
+    <h4>Courses</h4>
+    {% for c in courses %}
+        <li>
+            <a href="{% url 'course' course_id=c.id %}">{{c.title}} - {{c.number}}</a>
+        </li>
+    {% endfor %}
+</div>
+
+<hr>
+
+<div class="container">
+    <h4>Instructors</h4>
+    {% for i in instructors %}
+        <li>
+            <!-- TODO: create instructor page route -->
+            {{i.first_name}} {{i.last_name}}
```

Comment:
`{{ i.full_name }}` is equivalent

Reply:
Options
1. Index `full_name` but have redundancy in the index
2. Do `Instructor.objects.get(pk=id)` to get `full_name` but computationally expensive
3. Leave it as is

Reply:
Leave as is